reflex/components/radix/themes/components/radiogroup.py

```diff
@@ -73,3 +80,47 @@ class RadioGroupItem(CommonMarginProps, RadixThemesComponent):
 
     # When true, indicates that the user must check the radio item before the owning form can be submitted.
     required: Var[bool]
+
+
+def radio_group(
+    items: Var[list[str]],
+    direction: Var[LiteralFlexDirection] = "column",
+    gap: Var[LiteralSize] = "2",
+    **props
+) -> Component:
+    """Create a radio group component.
+
+    Args:
+        items: The items of the radio group.
+        direction: The direction of the radio group.
+        gap: The gap between the items of the radio group.
+        **props: Additional properties to apply to the accordion item.
+
+    Returns:
+        The created radio group component.
+    """
+
+    def radio_group_item(value: str) -> Component:
+        return Text.create(
+            Flex.create(
+                RadioGroupItem.create(value=value),
+                value,
+                gap="2",
```

Comment:
should this also use the passed `gap` for consistency?

Reply:
This is the `gap=2` is the distance between the radio group and the text it is associated with. The gap that we allow the user to pass is the distance between the different radio group items. I therefore do not think they should be passed the same value, and think we should keep the value between the radio group and the text fixed.
![Screenshot 2024-01-17 at 9 58 14 AM](https://github.com/reflex-dev/reflex/assets/64492814/fbccfd68-3e8e-46d8-b6b6-932cd8c73b4a)